recipe/meta.yaml

```diff
@@ -1,16 +1,15 @@
-{% set version = "3.5.3" %}
+{% set version = "4.3.4" %}
 
 package:
   name: redis-py
   version: {{ version }}
 
 source:
-  url: https://github.com/andymccurdy/redis-py/archive/{{ version }}.tar.gz
-  sha256: 0e3ef04af57d17207c2f13a49641a63b98e0040a10756a8247e833468b5a8206
+  url: https://github.com/redis/redis-py/archive/v{{ version }}.tar.gz
+  sha256: a3d3b6e86cc73c253fd5c02d174098abf3fa043dfdcb29689eb928f60a552fd6
 
 build:
   number: 0
-  noarch: python
```

Comment:
Should be skipped `py<36`, see https://github.com/redis/redis-py/blob/e6cd4fdf3b159d7f118f154e28e884069da89d7e/setup.py#L33